RPIPs/RPIP-committee_stipends.md

```diff
@@ -0,0 +1,130 @@
+type: Meta
+created: 2024-03-04
+requires: 10, 40
+---
+
+## Abstract
+
+This RPIP defines stipends for members of pDAO committees. The total stipend budget for a given committee is calculated based on the target number of members for that committee, the Target Hours for that committee, and the global stipend rate, which is the same for all committees. Committees may split their stipend budget amongst their members in any manner they choose. 
+
+Stipends to the members of each committee are paid by the Grants Management Committee.
+
+## Motivation
+This RPIP is primarily motivated by a desire to see the efforts being made by members of pDAO committees to be rewarded monetarily for the time they are committing to Rocket Pool as members of those committees. A secondary yet still important motivation is to help ensure that the pDAO continues to attract sufficient nominees for these committee membership roles in the future.
+
+Despite this motivation, there is also a desire to maintain a volunteer ethos among committee members. Compensation of any kind does threaten this ethos, however, it is hoped that by having financial compensation be closer to 'symbolic' than 'market rate' the pDAO can preserve this ethos. 
+
+## Specification
+
+### Stipends
+* A monthly stipend budget SHALL be allocated to each pDAO committee as defined in RPIP-10.
+* This stipend SHOULD be dispersed to voting committee members based on their contributions to the committee over the previous month. 
+* The stipend is denominated in USD.
+
+### Formula and Input Parameters
+Three input parameters affect each committee's total stipend budget.
+
+$TargetHours$ - Target Hours. The amount of hours the pDAO hopes each member commits to committee work each month. This parameter may be different for each committee.  
+
+$TargetCount$ - Target Member Count. The target number of members for the given committee. This parameter may be different for each committee.  
+
+$TargetRate$ - Global Stipend Rate. The hourly rate for time committed to a committee. This parameter is shared across all committees.  
+
+The committee stipend budget to be split amongst the committee members each month is defined by the following formula.
+```math
+$$
+
+Committee Monthly Stipend Budget = TargetHours * TargetCount * TargetRate
+
+$$
+```
+
+#### Parameter Changes
+
+* The Target Hours input for a given committee MAY be adjusted via the internal vote of that committee. 
+  * Increases to Target Hours MUST have a unanimous positive vote outcome, which MUST be posted publicly.
+  * Decreases to Target Hours MUST have a majority positive vote outcome, which MUST be posted publicly. 
+  * Any Target Hours change is subject to pDAO challenge, and SHALL NOT be acted upon until 1 calendar month has passed after the decision has been announced.
+* The Target Member Count SHALL be modified only by the pDAO via a voted modification to this RPIP and SHOULD follow the usual processes for such a modification. 
```

Comment:
Hmmm... 

For the GMC, https://rpips.rocketpool.net/RPIPs/RPIP-40 has `The GMC SHALL contain nine individuals, except in cases of vacancies.`

For the IMC, there is no such limitation. It's simply the size per the selection RPIP (though the IMC could boot folks with a 2/3 majority). RPIP-10 does establisht that all MCs are at least a 3-of-5.

I think having selection vote set this might be the move?

Reply:
> Might be simpler to just return it to current # of members.
This sounds like the move as it avoids the interactions with MC charters